package.json

```diff
@@ -81,7 +81,7 @@
   },
   "dependencies": {
     "bignumber.js": "7.2.1",
-    "eth-lightwallet": "^3.0.1",
+    "eth-lightwallet": "dakingha69/eth-lightwallet",
```

Comment:
whats this?

Reply:
In this cases please use a fork under our name. This temporary things usually last longer and then only you can make changes. 
https://github.com/trustlines-network/eth-lightwallet

Reply:
Just apply your commit